### ISSUE TITLE
Unknown constructor found 0xd597650c

### ISSUE THREAD

**Issue by zevlg** [tdlib/td#1603]
With TDLib 1.7.5 I've got this error:
```
[ 1][t 4][1625821811.703012943][UpdatesManager.cpp:896][#1][!UpdatesManager]	Receive updates.getDifference error: [Error : 500 : Unknown constructor found 0xd597650c]
```

while dealing with voice chats I believe, possibly as a result of `getGroupCall` call.  What does this mean?

Thanks

**Comment by levlam** [comment]
Pushed code for TDLib 1.7.5 accidentally used yet unstable server-side API. This will be fixed in a few days with TDLib 1.7.6. Alternatively, you can use TDLib from commit 12ae67a64f3a64e910fd8eb7192a117151e7f6d0 with critical fix bed2650b4c52b54a009a2df5d40be16497467f59 for secret chats creation.

**Comment by levlam** [comment]
It should be fixed in master.
